src/java.base/share/classes/java/io/ByteArrayOutputStream.java

```diff
@@ -282,9 +280,9 @@ public synchronized String toString(Charset charset) {
      * copied into it. Each character <i>c</i> in the resulting string is
      * constructed from the corresponding element <i>b</i> in the byte
      * array such that:
-     * <blockquote><pre>{@code
-     *     c == (char)(((hibyte & 0xff) << 8) | (b & 0xff))
-     * }</pre></blockquote>
+     * {@snippet lang=java :
+     *         c == (char)(((hibyte & 0xff) << 8) | (b & 0xff))
```

Comment:
I'd skip the extra indentation for just the single line example.

Reply:
So changed in b6abb2e40fe574f40ebdfc0c5b1e28ed4b09126e.